### PR TITLE
Separate QC global files for sync and async

### PR DESCRIPTION
**Pull request by knopers8**
@chiarazampolli

### CODE REVIEW

File: DATA/production/qc-workflow.sh [hunk 1]
```diff
@@ -38,10 +40,10 @@ if [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
     [[ -z "$QC_JSON_FT0" ]] && QC_JSON_FT0=$O2DPG_ROOT/DATA/production/qc-async/ft0.json
     [[ -z "$QC_JSON_FV0" ]] && QC_JSON_FV0=$O2DPG_ROOT/DATA/production/qc-async/fv0.json
     [[ -z "$QC_JSON_MID" ]] && QC_JSON_MID=$O2DPG_ROOT/DATA/production/qc-async/mid.json
-    [[ -z "$QC_JSON_GLO" ]] && QC_JSON_GLO=$O2DPG_ROOT/DATA/production/qc-async/glo.json
```
Comment (knopers8) [inline DATA/production/qc-workflow.sh:41]: This is a leftover that I forgot to remove previously, primtvx, itstpc and ittpctof replace this one.

File: DATA/production/qc-async/qc-global.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "host": "ali-qcdb.cern.ch:8083",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "REPLACE_ME_RUNNUMBER",
+        "type": "2",
+        "passName": "REPLACE_ME_PASS",
+        "periodName": "OCT",
```
Comment (knopers8) [inline DATA/production/qc-async/qc-global.json:15]: @chiarazampolli Perhaps we should also replace the periodName?

**Review by davidrohr — APPROVED** [review]
looks good to me

**Comment by chiarazampolli** [comment]
Hi, 

It looks fine to me too. Yes, let's replace also the period. 

Chiara

**Pull request merged**